src/core/packages/rendering/browser/README.mdx

```diff
@@ -0,0 +1,30 @@
+---
+id: kibDevRenderingService
+slug: /kibana-dev-docs/rendering-service
+title: Core Rendering Service
+description: This a service provided by SharedUX which abstracts the dependencies needed for a fully-loaded React context. 
+tags: ['shared-ux', 'core', 'react', 'context']
+date: 2023-07-25
+---
+
+## Description
+
+### Custom React elements require global context
+Kibana's browser-side architecture is an application composed of many different points where custom React elements can be mounted onto a DOM container. These elements have a self-contained component tree, and these and the process is often described as an "ad-hoc" render. Normally, a self-contained component would have no awareness of global context and core services, such as i18n and theming. That is why we need to wrap in a shared context providers, and somehow give those providers the service props they need.
```

Comment:
Was not entirely following this part, so I made some edits based on my assumptions.

```suggestion
Kibana's browser-side architecture is an application composed of many different points where custom React elements can be mounted onto a DOM container. These elements have a self-contained component tree, this process is often described as an "ad-hoc" render. A self-contained component tree has no awareness of global context or stateful core services, such as i18n and theming which are often needed. That is why we use shared context providers.
```